packages/forms/src/UIForm/fields/Text/Text.component.js

```diff
@@ -1,12 +1,24 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import { OverlayTrigger, Popover } from 'react-bootstrap';
+
 import FieldTemplate from '../FieldTemplate';
 import { generateDescriptionId, generateErrorId } from '../../Message/generateId';
-
 import { convertValue } from '../../utils/properties';
 
 export default function Text(props) {
-	const { id, isValid, errorMessage, onChange, onFinish, schema, value } = props;
+	const {
+		id,
+		isValid,
+		errorMessage,
+		onChange,
+		onFinish,
+		schema,
+		value,
+		overlayComponent,
+		overlayPlacement,
+	} = props;
```

Comment:
No need to spread the entire `props` object if most of the values are only used once.